### PR TITLE
Campaign Summary Page

### PR DESCRIPTION
**Pull request by Disquietude**
image

Campaign summary page that displays all the steps in a campaign. Can also add steps and edit step templates using the text editor (thanks @airensurzyn ). Requires material-ui version 4.9.10 (run `npm i` before running).

### CODE REVIEW

File: client/src/Views/Home/Campaigns/Campaign/CampaignSidebar.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          to={`/home/campaigns/${props.campaignId}/summary`}
+          disableRipple
+        />
+        <SidebarTab
+          label="Prospects"
+          value="prospects"
+          component={Link}
+          to={`/home/campaigns/${props.campaignId}/prospects`}
+          disableRipple
+        />
```
Comment (skassam21) [inline client/src/Views/Home/Campaigns/Campaign/CampaignSidebar.js:80]: small thing, but could use a map for these, especially as the tabs grow

**Review by skassam21 — COMMENTED** [review]
Looks great!

**Comment by skassam21** [comment]
I mentioned this to Airen too, but we should look at consolidating the StepEditor and TemplateEditor in the future since they have very similar components to them

File: client/src/Views/Home/Campaigns/Campaign/Step.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+const EmailTooltip = withStyles({
+  tooltip: {
+    fontSize: "0.8rem"
+  }
+})(Tooltip);
+
+const Step = props => {
+  const classes = useStyles();
+
+  const generateColumns = summary => {
```
Comment (airensurzyn) [inline client/src/Views/Home/Campaigns/Campaign/Step.js:49]: Nice implementation

**Review by airensurzyn — APPROVED** [review]
Looks good, danny. Layout looks very clean.

**Pull request merged**